VultisigApp/VultisigApp/Views/Components/BlowfishWarningInformationNote.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+
+struct BlowfishWarningInformationNote: View {
+    
+    @State var blowfishMessages: [BlowfishResponse.BlowfishWarning] = []
+    
+    var body: some View {
+        
+        if blowfishMessages.isEmpty {
+            
+            HStack(spacing: 12) {
+                icon
+                text
+            }
+            .padding(12)
+            .background(Color.green.opacity(0.35))
+            .cornerRadius(12)
+            .overlay(
+                RoundedRectangle(cornerRadius: 12)
+                #if os(iOS)
+                    .stroke(Color.green, lineWidth: 1)
+                #elseif os(macOS)
+                    .stroke(Color.green, lineWidth: 2)
+                #endif
+            )
+            
+        } else {
+            HStack(spacing: 12) {
+                icon
+                text
+            }
+            .padding(12)
+            .background(Color.warningYellow.opacity(0.35))
+            .cornerRadius(12)
+            .overlay(
+                RoundedRectangle(cornerRadius: 12)
+                #if os(iOS)
+                    .stroke(Color.warningYellow, lineWidth: 1)
+                #elseif os(macOS)
+                    .stroke(Color.warningYellow, lineWidth: 2)
+                #endif
+            )
+        }
+    }
+    
+    var icon: some View {
+        if blowfishMessages.isEmpty {
+            Image(systemName: "checkmark.shield")
+                .foregroundColor(Color.green)
+        } else {
+            Image(systemName: "exclamationmark.triangle")
+                .foregroundColor(Color.warningYellow)
+        }
+    }
+    
+    var text: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            
+            if blowfishMessages.isEmpty {
+                Text("Transaction Scanned Successfully! You are cleared to proceed with safety.") // Assuming `message` has a `message` property
```

Comment:
If the response from blowfish is empty , then please don't show anything , this message give user a sense guarantee , which is not good

Reply:
Shouldn’t we show we scanned the transaction? I can hide it as well.

Reply:
Well , or you can just say "Transaction scanned by Blowfish" , and make sure it is translated

Reply:
I will do that "Transaction scanned by Blowfish"

Reply:
<img width="464" alt="image" src="https://github.com/user-attachments/assets/0b08e3e1-1c08-4e86-8f66-5501538ab34b">

@johnnyluo